### ISSUE TITLE
[Refactor] Use relative imports for files in `lib`

### ISSUE THREAD

**Issue by a-mabe** [a-mabe/OpenHIIT#40]
```
// Don't
import 'package:demo/src/utils/dialog_utils.dart';

// Do
import '../../../utils/dialog_utils.dart';
```

**Comment by a-mabe** [comment]
Hi @Aryashakya, the help is much appreciated. Please follow the [contributing guidelines](https://github.com/a-mabe/OpenHIIT#contributing) and feel free to start work on the issue.

**Comment by a-mabe** [comment]
Closed in #49
